partner_contact_department/models/res_partner.py

```diff
@@ -0,0 +1,34 @@
+# © 2014-2015 Tecnativa S.L. - Jairo Llopis
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    department_id = fields.Many2one("res.partner.department", "Department")
+
+
+class ResPartnerDepartment(models.Model):
```

Comment:
I'm not sure, but we should following the rule: 1 class in 1 file.

Reply:
hhhmmm... I don't really understand what you mean. Could you give me more details?

Thx for reviewing!

Reply:
yes, exactly, one model class in one file

Reply:
Is it in the OCA guidelines? I mean... I don't mind changing this, but I haven't heard about that. THX!

Reply:
I've have seen it in Odoo 13 Survey Addon:

https://github.com/odoo/odoo/tree/13.0/addons/survey/models vs
https://github.com/odoo/odoo/tree/12.0/addons/survey/models

But this is no show stopper, :+1: